pkg/reconciler/apis/crdcleanup/crdcleanup_controller.go

```diff
@@ -80,33 +80,29 @@ func NewController(
 
 	crdInformer.Informer().AddEventHandler(cache.FilteringResourceEventHandler{
 		FilterFunc: func(obj interface{}) bool {
-			crd, ok := obj.(*apiextensionsv1.CustomResourceDefinition)
```

Comment:
Technically this is a regression in behavior but I think in practice it's the more correct thing to do anyway?

Reply:
Yes, but it's better to panic to indicate programmer error I'd say.